build.gradle.kts

```diff
@@ -94,15 +197,16 @@ mavenPublishing {
     val releaseVersion = project.findProperty("releaseVersion") as String? ?: System.getenv("VERSION")
```

Comment:
not really related to this PR, but I'm seeing:

```
> Configure project :
w: file:///.../build.gradle.kts:214:13: Name shadowed: releaseVersion
```

I'm assuming that's deliberate so that we don't pick up whatever `VERSION` is in the environment down below in `pom {`, but just sanity checking.

Reply:
I think that warning has been there for some time - in a follow up I'd like to make some changes here similar to changes in connect-kotlin to better choose release versions based on the most recent git tag, etc. I can look more at this error as part of that work.

Reply:
Fixed the shadowing error as there was another typo in the name. Verified that local release works after the changes.